### ISSUE TITLE
EM start year leads to different estimates

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [ss3sim/ss3sim#254]
Estimates are slightly different depending on when the EM is started relative to the first year of catch. Typically, I don't think a stock assessment model would be started before a year in which there are catches. In ss3sim we start the EM the same year as the OM, which has a 25 year burn in period. I think we should move to more standard stock assessment practices and start the EM in the first year of data rather than to match the OM. 
Replicate 1 started in year 1 and replicate 2 started in year 26. Estimates are slightly different, but the desire to change is based on actual practice rather than differences in this toy example.
![EMstartyear](https://user-images.githubusercontent.com/4108564/62719101-1d6a3280-b9bc-11e9-9d12-2848e72a39bc.png)

**Comment by k-doering-NOAA** [comment]
I think that makes sense to change the default start year for the EM to be in the first year of catch to better align with more realistic stock assessment practices. And the estimates look pretty similar to me, anyway. 

This would be a change in the cod-EM model external data files, not done by manipulating copies of the model in ss3sim, correct?

**Comment by kellijohnson-NOAA** [comment]
No, there is no EM dat file. It would be done internally within ss3sim based on the first year of catches or the first year of survey data. One would need to remove zero catches, change the start year of the model, and fix the rec dev estimation portion of the control file, and finally check the forecast file. We have a `change_year` function, but I haven't looked at it in a while to know if that would work for this application.

**Comment by k-doering-NOAA** [comment]
@kellijohnson-NOAA this slipped off my radar - is this something we want to change relatively soon (to make ss3sim more realistic to real stock assessment practices), or something we want to label as an enhancement for the future?

**Comment by k-doering-NOAA** [comment]
This is how ss3sim is working now, so this issue closed. Note, however, that a user may run into bugs if not using the default 25 years of burn in (#282)

**Comment by k-doering-NOAA** [comment]
This is implemented, but needs to be documented in the vignette.

**Comment by kellijohnson-NOAA** [comment]
Via email, Merrill asked for the ability of the user to set a desired start year of the model.
